front/src/app/admin-users/admin-users.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { DataService } from '../services/data.service';
+
+@Component({
+  selector: 'app-admin-users',
+  templateUrl: './admin-users.component.html',
+  styleUrls: ['./admin-users.component.css']
+})
+export class AdminUsersComponent implements OnInit {
+
+  userList= [
+    {id: 1, username: 'user 1', created: '01-01-2019', type: 'admin'}, {id: 2, username: 'user 2', created: '01-01-2019', type: 'admin'},
+    {id: 3, username: 'user 3', created: '01-01-2019', type: 'user'}, {id: 4, username: 'user 4', created: '01-01-2019', type: 'user'},
+    {id: 5, username: 'user 5', created: '01-01-2019', type: 'user'}, {id: 6, username: 'user 6', created: '01-01-2019', type: 'user'},
+    {id: 7, username: 'user 7', created: '01-01-2019', type: 'user'}, {id: 8, username: 'user 8', created: '01-01-2019', type: 'user'},
+    {id: 9, username: 'user 9', created: '01-01-2019', type: 'user'}, {id: 10, username: 'user 10', created: '01-01-2019', type: 'user'},
+    {id: 11, username: 'user 11', created: '01-01-2019', type: 'user'}, {id: 12, username: 'user 12', created: '01-01-2019', type: 'user'},
+    {id: 13, username: 'user 13', created: '01-01-2019', type: 'user'}, {id: 14, username: 'user 14', created: '01-01-2019', type: 'user'},
+    {id: 15, username: 'user 15', created: '01-01-2019', type: 'user'}, {id: 16, username: 'user 16', created: '01-01-2019', type: 'user'}
+  ];
```

Comment:
Powinniście takie dane testowe wyrzucić do jakiegoś innego pliku, a najlepiej zrobić już jakiś serwis, który miałby się komunikować z backendem i zwracać z niego te dane testowe, żeby poudawać, że frontend gada z backendem. Przepięcie praktycznie gotowego serwisu ze zwracania danych testowych na faktyczne dane z backendu będzie już mało pracochłonne, a będziecie już mieli to praktycznie gotowe i małe będzie prawdopodobieństwo, że jakieś cuda zaczną się dziać, które zafundują wam kolejne godziny debugowania i szperania po stackoverflow.  
Jak będziecie chętni, mogę wam pokazać jak to rozwiązać z użyciem ngxs i angularowego klienta http, ale powiedzcie na początku, że chcecie żebyśmy sobie te 5-10 minut zarezerwowali na to.

Reply:
Na razie to tworzymy bazę dopiero i cały czas się zmienia jej struktura. Logowanie np już od początku jest przez http. Nie chciałem tworzyć funkcji pobierania z bazy nie znając nawet odpowiedniego adresu. Z tego co czytalem o ngxs to zwraca on Observable na daną, czy to działa to tak samo jak się chce połączyć async od firebase'a?